src/solvers/prop/prop.h

```diff
@@ -116,6 +116,11 @@ class propt:public messaget, public prop_assignmentt
     warning() << "CPU limit ignored (not implemented)" << eom;
   }
 
+  /// Set Random variable frequency (Exclusive to minisat2)
+  virtual void random_var_freq(std::size_t i) { }
+  /// Set Random variable seed (Exclusive to minisat2)
```

Comment:
Also glucose supports both settings. I suggest to add a has_random_seed() etc so that cbmc_solvers can check whether the selected solver supports the feature and can output an appropriate error message.